react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -124,18 +123,20 @@ class Thumbnail extends Component<Props> {
         const {
             _audioTrack: audioTrack,
             _isEveryoneModerator,
-            _isModerator,
             _largeVideo: largeVideo,
             _onClick,
             _onShowRemoteVideoMenu,
             _styles,
             _videoTrack: videoTrack,
-            disablePin,
             disableTint,
             participant,
-            renderDisplayName
+            renderDisplayName,
+            tileView
         } = this.props;
 
+        // Currently only tile view disables pinning, but there can be mroe later.
```

Comment:
I wonder if there really will be mroe later.

Reply:
Good point. I just wanted to stay consistent with the variable names, but probably you're right and it's not needed